openedx/core/djangoapps/credit/partition_schemes.py

```diff
@@ -0,0 +1,126 @@
+"""
+Provides partition support to the user service.
+"""
+
+import logging
+
+from course_modes.models import CourseMode
+from lms.djangoapps.verify_student.models import SkippedReverification, VerificationStatus
+from student.models import CourseEnrollment
+
+
+log = logging.getLogger(__name__)
+
+
+class VerificationPartitionScheme(object):
+    """
+    This scheme randomly assigns users into the partition's groups.
+    """
+    NON_VERIFIED = 'non_verified'
+    VERIFIED_ALLOW = 'verified_allow'
+    VERIFIED_DENY = 'verified_deny'
```

Comment:
Please add detailed comments explaining these different groups.  (You can re-use the descriptions from the ICRV access control proposal).